styles/globals.css

```diff
@@ -340,4 +340,8 @@ abbr[title] {
 
 .meeting-card:hover > div:nth-child(2) > p  {
   color: white;
+}
+
+code{
```

Comment:
Why have you made font-weight `bold` here?

Reply:
Hi @akshatnema,

The \<code/> content are not highlighted well and the users cannot identify the keywords used in the docs.
![image](https://github.com/asyncapi/website/assets/70331875/456e62c8-b6aa-4c95-9de1-d035616c282f)